data/avro-schema/src/main/resources/avro/net/corda/data/uniqueness/UniquenessCheckExternalRequest.avsc

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
As we are making breaking API changes, I'm assuming you will need to bump the platform API version, as per this process: https://r3-cev.atlassian.net/wiki/spaces/CB/pages/4071850170/Working+with+corda-api+and+corda-runtime-os+repos+for+Corda+5#What-do-I-need-to-do-to-merge-a-breaking-change-in-corda-api%3F